tests/shim_test.go

```diff
@@ -66,7 +75,9 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
-func (rig *testRig) Start() {
+func (rig *testRig) Start(t int) {
```

Comment:
This should take a `tokenType` instead of an `int`.
And call this variable tType or tokenT or anything else, but I think `t` can be confusing with testing.T handler.